mlkem/poly.h

```diff
@@ -570,6 +570,7 @@ ASSIGNS(OBJECT_WHOLE(r))
 ENSURES(ARRAY_IN_BOUNDS(int, k, 0, MLKEM_N - 1, r->coeffs, -3 * HALF_Q + 1, 3 * HALF_Q - 1));
 // clang-format on
 
+// clang-format off
```

Comment:
Can you explain why you moved the `//clang-format off`? Do you want to change it everywhere? 
If I see correctly then this basically means turning off clang-format for all headers files entirely.

Reply:
I personally find this easier to read, but I agree that we ultimately disable clang format this way. Let's not try to solve this in this PR.